Automattic/sensei
# Add or update documentation comments for action hooks

## Proposed Changes
*  Add or update documentation comments for action hooks.


## Testing Instructions
* Run `npm run build:docs`
* Run `open hookdocs/index.html`
* Check you see documentation for actions.

## Pre-Merge Checklist
